src/feeds/FeedReader.ts

```diff
@@ -189,7 +188,7 @@ export class FeedReader {
         log.debug('Loaded feed URLs:', this.observedFeedUrls);
 
         void this.loadSeenEntries().then(() => {
-            return this.pollFeeds();
+            Promise.all(Array.from({length: config.pollConcurrency}, (_, i) => this.pollFeeds(i)))
```

Comment:
As long as nothing is awaiting/thenning on `Promise.all`, could this just be a loop of `this.pollFeeds` calls that neither returns nor awaits?
```suggestion
            for (let i = 0; i < config.pollConcurrency; i++) {
                this.pollFeeds(i);
            }
```

Reply:
Yeah, can do. The previous code returned a promise so I felt it was right to, but it's not like we're using it!